include/triton/Dialect/TritonGPU/Transforms/Passes.td

```diff
@@ -9,6 +9,14 @@ def TritonGPUPipeline : Pass<"tritongpu-pipeline", "mlir::ModuleOp"> {
   let description = [{
     Applies software pipelining to loops in the module based on number of stages.
     This may convert some load into asynchronous loads, and multi-buffer the data.
+    The underlying idea is to hide data loading by pipelining loops,
+    basically, one stage to load/fetch data, a second one to compute the data and
```

Comment:
The description seems wrong to me

Reply:
Thanks for your feedback. This additional description has been removed from the PR.